papaya-web/src/main/java/pl/edu/agh/papaya/rest/projects/service/SprintsRestService.java

```diff
@@ -170,23 +179,30 @@ private SprintSummaryDto createSprintSummaryDto(List<UserAvailabilityDto> userAv
             );
 
             @SuppressWarnings({"MultipleStringLiterals"})
```

Comment:
Suppress potrzebny? `"finalTimePlanned"` ma jedno wystąpienie w pliku, a to chyba o to chodzi

Reply:
sprawdze